docs/_documentations/troubleshooting.md

```diff
@@ -231,3 +232,71 @@ Issue link:
 If you turn off `auto build` for a Node.js project when you run Codewind locally, it has no effect. Changes you make to your code automatically start or restart a build even though `auto build` is disabled.
 
 ***
+# Appsody with Codewind
+
+<!--
+Action/Topic: Appsody with Codewind
+Issue type: bug/info
+Issue link: https://github.com/eclipse/codewind-docs/issues/64
+18.10:
+-->
+## Node.js and Swift templates remain in the starting state
+Node.js and Swift templates remain in the starting state because they do not connect with the server.
```

Comment:
"they do not connect with the server" sounds a bit confusing.  It's because those projects don't start up a server for Codewind to ping